templates/includes/RequestItems_email.ss

```diff
@@ -0,0 +1,40 @@
+<table title="<%t DMSCart.TABLE_TITLE "Requested documents" %>">
+    <thead>
+    <tr>
+        <th>
+            <%t DMSCart.TITLE "Title" %>
+        </th>
+        <th>
+            <%t DMSCart.FILE_NAME "File name" %>
+        </th>
+        <th>
+            <%t DMSCart.TOTAL_COPIES "# of copies" %>
+        </th>
+        <th>
+            <%t DMSCart.TYPE "Type" %>
+        </th>
+        <th>
+            <%t DMSCart.SIZE "Size" %>
+        </th>
+        <th>
+            <%t DMSCart.URL "URL" %>URL
+        </th>
+    </tr>
+    </thead>
+    <tbody>
+        <% loop $Items %>
+        <tr>
+            <% with $Document %>
+                <td>$Title</td>
+                <td>$FilenameWithoutID</td>
+                <td>$Up.Quantity</td>
+                <td>$Extension.UpperCase</td>
+                <td>$Size</td>
+                <td>
+                    <a href="{$AbsoluteBaseURL}DMSCart/{$ID}"><%t DMSCart.VIEW_DOCUMENT "View Document" %></a>

```

Comment:
Does this link work? should be `dmsdocument/`? Please use `DMSDocument::getLink` instead of `$ID`